python/lsst/summit/utils/butlerUtils.py

```diff
@@ -385,19 +385,19 @@ def getExpIdFromDayObsSeqNum(butler: dafButler.Butler, dataId: dict) -> dict:
     return {"exposure": expRecord.id}
 
 
-def updateDataIdOrDataCord(dataId: dict, **updateKwargs: Any) -> dict:
+def updateDataIdOrDataCord(dataId: dafButler.DataId, **updateKwargs: Any) -> Mapping[str, Any]:
```

Comment:
`dataId` here can definitely also be a dict, and usually is, I think. Please make it be allowed to be both.

Reply:
DataId definition include also a Mapping, dict is also a Mapping and in the use on this function the Mapping is enough, but I can change it to dict (more general)